data-sets-api-server/src/main/java/org/zowe/unix/files/controller/UnixFilesController.java

```diff
@@ -112,8 +112,9 @@ private String getPathFromRequest(HttpServletRequest request) {
     notes = "This API deltes a Unix file. Try it out function will not work due to the encoding of forward slashes, "
             + "it should be noted that requests to this endpoint should only contain unencoded slashes", tags = "Unix Files APIs")
     @ApiResponses({ @ApiResponse(code = 204, message = "Unix file successfully deleted") })
-    public ResponseEntity<?> deleteUnixFile(@PathVariable String path, HttpServletRequest request
-    		, @RequestHeader(value = "X-IBM-Option", required = false) String recursiveOption) {
+    public ResponseEntity<?> deleteUnixFile(
+            @PathVariable String path, HttpServletRequest request,
+            @RequestHeader(value = "X-IBM-Option", required = false) String recursiveOption) {
```

Comment:
X-IBM-Option isn't a very intuitive name

Reply:
changed to X-IBM-Option to "recursive" boolean header